tests/integration/test_hedged_requests/test.py

```diff
@@ -418,10 +426,17 @@ def test_async_query_sending(started_cluster):
     )
     check_if_query_sending_was_not_suspended()
 
-    NODES["node"].query(
-        "SELECT hostName(), id FROM distributed_query_sending ORDER BY id LIMIT 1 SETTINGS"
-        " prefer_localhost_replica = 0, async_query_sending_for_remote=1, max_threads = 1, max_distributed_connections=1"
-    )
-    check_if_query_sending_was_suspended()
+    attempt = 0
+    while attempt < 1000:
```

Comment:
Isn't 1000 to much? maybe 100 is enough?

Reply:
Yes, I probably overdid it a little, 100 should be enough